redmine_zulip/lib/zulip_hooks.rb

```diff
@@ -105,10 +105,7 @@ def zulip_port()
    end
 
    def zulip_api_basename()
-     if Setting.plugin_redmine_zulip[:zulip_server]["api.zulip.com"]
-       return ""
-     end
-     return "/api"
+     return (Setting.plugin_redmine_zulip[:zulip_server].sub %r{^https?:(//|\\\\)(www\.)?}i, '').concat('/api')
```

Comment:
I saw that in all of our integrations we sent a request to "api.zulip.com" not to "https://api.zulip.com" not sure if its important but I'd like to escape the protocol part